.github/workflows/build.yml

```diff
@@ -374,7 +374,7 @@ jobs:
       with:
         path: ./hypothesis
         key: hypothesis-database-${{ github.head_ref || github.run_id }}
-        restore-keys:
+        restore-keys: |
           - hypothesis-database-
```

Comment:
@Zac-HD did you mean to keep that leading hyphen?

Reply:
This makes it look up the `'- hypothesis-database-'` prefix and I'm pretty sure you wanted `'hypothesis-database-'`:
```console
Cache not found for input keys: hypothesis-database-9927179210, - hypothesis-database-
```
*(https://github.com/python/cpython/actions/runs/9927179210/job/27421821417#step:21:24)*

Reply:
@Zac-HD here's the fix: https://github.com/python/cpython/pull/121756